ProteoformSuiteInternal/Lollipop.cs

```diff
@@ -184,6 +189,103 @@ public void process_neucode_components(List<NeuCodePair> raw_neucode_pairs)
+                Loaders.LoadElements(dir + @"\elements.dat");
+                var rawFile = ThermoDynamicData.InitiateDynamicConnection(f.complete_path);
+                Dictionary<int, Feature> featureIdToFeature = new Dictionary<int, Feature>();
+                StreamReader reader = new StreamReader(Path.Combine(Path.GetDirectoryName(f.complete_path), Path.GetFileNameWithoutExtension(f.complete_path)) + "_ms1ft.csv");
+                string line;
+                int lineNum = 1;
+
+                while (reader.Peek() > 0)
+                {
+                    line = reader.ReadLine();
+
+                    if (lineNum != 1)
+                    {
+                        var massWithScan = new Envelope(line, type);
+                        massWithScan.retentionTime = rawFile.GetOneBasedScan(massWithScan.scan_num).RetentionTime;
+
+                        if (featureIdToFeature.ContainsKey(massWithScan.FeatureID))
+                            featureIdToFeature[massWithScan.FeatureID].AddEnvelope(massWithScan);
+                        else
+                            featureIdToFeature.Add(massWithScan.FeatureID, new Feature(massWithScan));
+                    }
+
+                    lineNum++;
+                }
+
+                reader.Close();
+
+                string headerForThisFile = "No.\tMonoisotopic Mass\tSum Intensity\tNumber of Charge States\tNumber of Detected Intervals\tDelta Mass\tRelative Abundance\tFractional Abundance\tScan Range\tRT Range\tApex RT";
+
+                List<string> output = new List<string>() { headerForThisFile };
+                foreach (var feature in featureIdToFeature)
+                    if (feature.Value.monoisotopicMass > minMass && feature.Value.isotopicEnvelopes.Select(p => p.charge).Distinct().Count() >= numChargesRequired)
+                        output.Add(feature.Value.GetThermoFormattedString());
+
+                File.WriteAllLines(Path.Combine(Path.GetDirectoryName(f.complete_path), Path.GetFileNameWithoutExtension(f.complete_path)) + "_deconv.tsv", output);
+
+                Application excel = new Application();
+                Workbook wb = excel.Workbooks.Open(Path.Combine(Path.GetDirectoryName(f.complete_path), Path.GetFileNameWithoutExtension(f.complete_path)) + "_deconv.tsv");
+                excel.DisplayAlerts = false;
+                wb.SaveAs(Path.Combine(Path.GetDirectoryName(f.complete_path), Path.GetFileNameWithoutExtension(f.complete_path)) + "_deconv.xlsx", XlFileFormat.xlOpenXMLWorkbook);
+                excel.DisplayAlerts = true;
+                wb.Close();
+                excel.Quit();
+            }
+
+            //foreach (InputFile f in input_files.Where(f => f.purpose == Purpose.RawFile))
+            //{
+            //    string filepath = Path.GetDirectoryName(f.complete_path);
+            //    string filename = Path.GetFileNameWithoutExtension(f.complete_path);
+            //}
```

Comment:
for each of the files that promex outputs do
if(file.exists(FILEPath)) file.delete(FILEPath)